app/services/idv/actions/verify_document_status_action.rb

```diff
@@ -90,6 +99,11 @@ def async_state
         proofing_job_result
       end
 
+      def remaining_attempts
+        return nil unless document_capture_session
+        Throttler::RemainingCount.call(document_capture_session.user_id, :idv_acuant)
```

Comment:
Should this be using `verify_document_capture_session` ? Based on the logic of that function, it would appear to be accounting for the hybrid flow, which seems like what we'd want.

And if we do, is there ever a case that it's nil?

Reply:
Good catch - and yes? I think it technically _could_ be nil? Done in 49f940040